src/Runner.Worker/Handlers/ScriptHandler.cs

```diff
@@ -23,109 +23,113 @@ public sealed class ScriptHandler : Handler, IScriptHandler
 
         public override void PrintActionDetails(ActionRunStage stage)
         {
-            if (stage == ActionRunStage.Post)
+            // We don't want to display the internal workings if composite (similar/equivalent information can be found in debug)
+            if (!ExecutionContext.InsideComposite)
```

Comment:
All I did was wrap all the existing code in this if statement

Reply:
add the `if` check to the caller of `PrintActionDetails` to reduce diff?

Reply:
or should we print all these info using `ExecutionContext.Debug()` instead of `ExecutionContext.Output()`?